pkg/kubeapiserver/options/options.go

```diff
@@ -22,3 +22,6 @@ import (
 
 // DefaultServiceNodePortRange is the default port range for NodePort services.
 var DefaultServiceNodePortRange = utilnet.PortRange{Base: 30000, Size: 2768}
+
+// DefaultServiceIPCIDR is a CIDR notation of IP range from which to allocate service cluster IPs
+var DefaultServiceIPCIDR = "10.0.0.0/24"
```

Comment:
You can make this a const, or (better) make this store the parsed form. It's one thing if ignoring the error is right next to the place where it's declared const, another if ignoring the error is distributed into other places. Don't want to encourage that.

Reply:
Let me try understand your suggestion. If we do this (the parsed form):

```
 var DefaultServiceIPNet net.IPNet = net.IPNet{IP: net.ParseIP("10.0.0.0"), Mask: net.CIDRMask(24, 32)}
```

We then can use `DefaultServiceIPNet` without having to worry about its format?

Reply:
Yes. Sorry for delay, I am way behind on reviews.

Reply:
Done.